src/DOMTokenList.js

```diff
@@ -23,7 +23,10 @@ module.exports = class DOMTokenList extends Array {
   }
 
   contains(token) {
-    return this.includes(token);
+    for (let i = 0, n = this.length; i < n; i++) {
```

Comment:
That's a glaring oversight. I'll be sure to refer to the reference implementation. Thanks for pointing it out!